src/jvm/test/org/jetbrains/kotlinx/lincheck/test/FailedScenarioMinimizationTest.kt

```diff
@@ -27,18 +27,20 @@ class FailedScenarioMinimizationTest: VerifierState() {
 
     @Test
     fun testWithoutMinimization() {
-        val options = StressOptions().actorsPerThread(10)
-                                     .invocationsPerIteration(100_000)
-                                     .minimizeFailedScenario(false)
+        val options = StressOptions()
+            .actorsPerThread(10)
+            .invocationsPerIteration(100_000)
+            .minimizeFailedScenario(false)
         try {
             LinChecker.check(FailedScenarioMinimizationTest::class.java, options)
-            fail("Should fail with AssertionError")
-        } catch (e: AssertionError) {
-            val m = e.message!!
-            assertTrue("The init part should NOT be minimized", m.contains("Init"))
-            assertTrue("The post part should NOT be minimized", m.contains("Post"))
-            assertEquals("The parallel part should NOT be minimized",
-                    10, m.lines().filter { it.contains("|") }.size)
+            fail("Should fail with LincheckAssertionError")
+        } catch (e: LincheckAssertionError) {
+            val failedScenario = e.failure.scenario
+            assertTrue("The init part should NOT be minimized", failedScenario.initExecution.isNotEmpty())
```

Comment:
For this test it is impossible, because the test uses stress-strategy and fails non-deterministically with some unknown execution result (and thus the output is non-deterministic)